x/emissions/keeper/keeper.go

```diff
@@ -203,6 +214,11 @@ type Keeper struct {
 
 	topicLastWorkerCommit  collections.Map[TopicId, types.TimestampedActorNonce]
 	topicLastReputerCommit collections.Map[TopicId, types.TimestampedActorNonce]
+
+	// active reputers for a topic
+	activeReputers collections.KeySet[collections.Pair[TopicId, ActorId]]
```

Comment:
all of these need export and import in `x/emissions/keeper/genesis.go` so we can dump database state in an emergency

Reply:
Updated!